files/templates/alertmanager/alertmanager.yaml

```diff
@@ -345,7 +345,7 @@ receivers:
 
 inhibit_rules:
 - source_matchers:
-  - kube_state_metrics_down=true
+  - inhibit_kube_state_metrics_down=true
```

Comment:
Do you have the related PR on prometheus-rules for this change ?

Reply:
https://github.com/giantswarm/prometheus-rules/pull/733#discussion_r1182352088

Reply:
Why doesn't the prometheus-rules PR set `cancel_if_inhibit_kube_state_metrics_down` everywhere `cancel_if_kube_state_metrics_down` is set ?

Reply:
Because we decided to use cancel_if_kube_state_metrics_down and not cancel_if_inhibit_kube_state_metrics_down right?

Reply:
I'm loosing my mind...